src/main.js

```diff
@@ -85,6 +89,14 @@ function clearInputs() {
   inputTitle.value = '';
 }
 
+function disableSaveButton(event) {
+  if (event.target.parentNode.childNodes[3].value !== ''
```

Comment:
Id like to dive a little deeper on this line and its proceeding line. I understand that we're targeting a position, but I'm not quite sure what the exact position is.